tests/test_common_gpu.py

```diff
@@ -79,8 +80,7 @@
         from peft.tuners.vera import Linear4bit as VeraLinear4bit
 
 
-@require_non_cpu
```

Comment:
I don't think we can just remove this decorator, since this class contains a bunch of tests that are unrelated to gptqmodel. From my understanding, gptqmodel can be run on CPU. In this case, I would suggest to move the gptqmodel tests to a different test file and leave this class untouched otherwise. They can be combined there with the `PeftGPTQTests` tests from `test_gpu_examples.py`.

The new test file (e.g. `tests/test_gptqmodel.py`) can then be run during the normal GitHub CI, which will just use CPU. However, if the tests are very slow on CPU, we need to either accelerate them or not run them on normal CI after all.